### DISCUSSION TITLE
Non-profit ticket not answered #2609298

### DISCUSSION THREAD

**Discussion by itsatgithub** [community/community#119954] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, We've submitted a ticket to support requesting non profit plan on 20/2/24 and anyone contact us yet, I imagine you guys are overstuffed with work but it's been 2 month without any answer.

**Comment by ebndev** [comment]
Hi @itsatgithub,

Thank you for your patience. This community is managed by GitHub employees, but it is not a support forum. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
